site/src/pages/DeploymentSettingsPage/LicensesSettingsPage/LicenseSeatConsumptionChart.tsx

```diff
@@ -0,0 +1,238 @@
+import { Button } from "components/Button/Button";
+import {
+	type ChartConfig,
+	ChartContainer,
+	ChartTooltip,
+	ChartTooltipContent,
+} from "components/Chart/Chart";
+import {
+	Collapsible,
+	CollapsibleContent,
+	CollapsibleTrigger,
+} from "components/Collapsible/Collapsible";
+import { Link } from "components/Link/Link";
+import { Spinner } from "components/Spinner/Spinner";
+import { ChevronRightIcon } from "lucide-react";
+import type { FC } from "react";
+import { Link as RouterLink } from "react-router-dom";
+import {
+	Area,
+	AreaChart,
+	CartesianGrid,
+	ReferenceLine,
+	XAxis,
+	YAxis,
+} from "recharts";
+
+const chartConfig = {
+	users: {
+		label: "Users",
+		color: "hsl(var(--highlight-green))",
+	},
+} satisfies ChartConfig;
+
+export type LicenseSeatConsumptionChartProps = {
+	limit: number | undefined;
+	data:
+		| {
+				date: string;
+				users: number;
+		  }[]
+		| undefined;
+};
+
+export const LicenseSeatConsumptionChart: FC<
+	LicenseSeatConsumptionChartProps
+> = ({ data, limit }) => {
+	return (
+		<section className="border border-solid rounded">
+			<div className="p-4">
+				<Collapsible>
+					<header className="flex flex-col gap-2 items-start">
+						<h3 className="text-md m-0 font-medium">
+							License seat consumption
+						</h3>
+
+						<CollapsibleTrigger asChild>
+							<Button
+								className={`
+									h-auto p-0 border-0 bg-transparent font-medium text-content-secondary
+									hover:bg-transparent hover:text-content-primary
+									[&[data-state=open]_svg]:rotate-90
+								`}
+							>
+								<ChevronRightIcon />
+								How we calculate license seat consumption
+							</Button>
+						</CollapsibleTrigger>
+					</header>
+
+					<CollapsibleContent
+						className={`
+							pt-2 pl-7 pr-5 space-y-4 font-medium max-w-[720px]
+							text-sm text-content-secondary
+							[&_p]:m-0 [&_ul]:m-0 [&_ul]:p-0 [&_ul]:list-none
+						`}
+					>
+						<p>
+							Licenses are consumed based on the status of user accounts. Only
+							Active user accounts are consuming license seats.
+						</p>
+						<ul>
+							<li className="flex items-center gap-2">
+								<div
+									className="rounded-[2px] bg-highlight-green size-3 inline-block"
+									aria-label="Legend for active users in the chart"
+								/>
+								The user was active at least once during the last 90 days.
+							</li>
+							<li className="flex items-center gap-2">
+								<div
+									className="size-3 inline-flex items-center justify-center"
+									aria-label="Legend for license seat limit in the chart"
+								>
+									<div className="w-full border-b-1 border-t-1 border-dashed border-content-disabled" />
```

Comment:
```suggestion
									<div className="w-full border-b border-t border-dashed border-content-disabled" />
```
this reduces the width to better match the design